Cabal/Distribution/Utils/Safe.hs

```diff
@@ -0,0 +1,6 @@
+module Distribution.Utils.Safe where
```

Comment:
Since `Distribution.Simple.Utils` already imports `Distribution.Utils.Glob`, to avoid circular imports, I added `tailMay` to this new module, and then also re-exported it from `Distribution.Simple.Utils`. Is that ok?